swo_aws_extension/flows/steps/create_new_aws_environment.py

```
@@ -0,0 +1,111 @@
+                "%s - Warning - Master Payer Account ID is missing in the order parameters",
+                context.order_id,
+            )
+            context.order = set_ordering_parameter_error(
+                context.order,
+                OrderParametersEnum.MASTER_PAYER_ACCOUNT_ID,
+                ERR_MISSING_MPA_ID.to_dict(),
+            )
+            raise QueryStepError(
+                f"{context.order_id} - Querying - Querying due to missing Master Payer Account ID.",
+                OrderQueryingTemplateEnum.NEW_ACCOUNT_CREATION,
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)